src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -1069,8 +1069,8 @@ export default class ReimbursementRequestService {
    * @param organizationId the organization the user is currently in
    * @returns a buffer of the image data and the image type
    */
-  static async downloadReceiptImage(fileId: string, submitter: User, organization: Organization) {
-    await validateUserIsPartOfFinanceTeamOrAdmin(submitter, organization.organizationId);
+  static async downloadReceiptImage(fileId: string, _submitter: User, _organization: Organization) {
+    // await validateUserIsPartOfFinanceTeamOrAdmin(submitter, organization.organizationId);
```

Comment:
Oh yeah so the problem I was having was that this method only allows finance or admins to be able to download an image, i was going to ask if this permissions is necessary because everyone should be able to download images. In that case I could change the name of the function and just remove that permissions check

Reply:
Ah I see, I think you should just leave it as it was though

Reply:
Wait so should I make a new function then or how should I do it?

Reply:
Yes this is just for downloading receipt images, should be separate function for downloading unrelated images.